### PR TITLE
feat: add realtime to trip details

### PR DESCRIPTION
**Pull request by ingvisim**
Add realtime with previously passed bus stop on trip details.

<details>
<summary>Screenshots</summary>

Simulator Screen Shot - iPhone 14 Pro - 2022-12-13 at 13 38 56
Simulator Screen Shot - iPhone 14 Pro - 2022-12-13 at 13 38 31
</details>

https://github.com/AtB-AS/atb-bff/pull/197

Close https://github.com/AtB-AS/kundevendt/issues/2949

### CODE REVIEW

File: src/translations/screens/subscreens/TripDetails.ts [hunk 1]
```diff
@@ -11,7 +11,11 @@ const TripDetailsTexts = {
           `Steg ${stepNumber}, ${travelMode}`,
           `Step ${stepNumber}, ${travelMode}`,
         ),
-
+      lastPassedStop: (stopPlaceName: string, time: string) =>
+        _(
+          `Passerte ${stopPlaceName} kl. ${time}`,
+          `Passed ${stopPlaceName} at ${time}`,
+        ),
```
Comment (gorandalum) [inline src/translations/screens/subscreens/TripDetails.ts:18]: Should be more precise regarding `StopPlace` or `Quay`. Here the parameter is named `stopPlaceName`, but you are sending in `quay.name`.

File: src/screens/TripDetails/components/TripSection.tsx [hunk 2]
```diff
@@ -168,7 +172,25 @@ const TripSection: React.FC<TripSectionProps> = ({
             />
           </TripRow>
         )}
-
+        {lastPassedStop?.quay?.name && (
+          <TripRow>
```
Comment (gorandalum) [inline src/screens/TripDetails/components/TripSection.tsx:176]: Should this be clickable? Kinda hard to know from the sketches. It seems kinda "grouped together" with the other clickable text like the "3 intermediate stops".

**Review by gorandalum — APPROVED** [review]
Looks good! Some comments you can look at.

File: src/screens/TripDetails/components/TripSection.tsx (hunk 2 above)
Comment (rosvik) [reply src/screens/TripDetails/components/TripSection.tsx:176]: I know we have talked about this before, that some people have a hard time finding the part that is clickable. I feel like we should make the line name clickable as well. Maybe @ckbilstad have an opinion? (Which part of a leg in trip details should be clickable, and lead to departure details?)

Comment (gorandalum) [reply src/screens/TripDetails/components/TripSection.tsx:176]: I agree, it's hard to find. Maybe adding an arrow or something would help as well.

Comment (ckbilstad) [reply src/screens/TripDetails/components/TripSection.tsx:176]: I agree. Dont have the time to jump into it right now tho. Mention it to Sebastian or create an issue so we can pick it up again soon.

Comment (rosvik) [reply src/screens/TripDetails/components/TripSection.tsx:176]: https://github.com/AtB-AS/kundevendt/issues/3066

**Pull request merged**